aie_runtime_lib/AIE2/lut_based_ops.h

```diff
@@ -22,7 +22,8 @@ alignas(aie::vector_decl_align) extern int16 exp_flut_ab[512];
 alignas(aie::vector_decl_align) extern int16 exp_flut_cd[512];
 alignas(aie::vector_decl_align) extern unsigned char m_inv_lut[128];
 
-__attribute__((always_inline)) v16accfloat getExpBf16(v16bfloat16 x) {
+static inline __attribute__((always_inline)) v16accfloat
```

Comment:
not a good idea to put static functions into a header file, even if they are marked inline